examples/error-subtyping/error_subtyping.md

```diff
@@ -1,9 +1,12 @@
 # Error subtyping
 
-`distinct` creates a new subtype and can be used to define subtypes of `error`. The name of the distinct `error` type can be used with the error constructor to create an `error` value of that type.
+A given non-distinct `error` type (such as `InvalidIntDetail` in the example) is a supertype of another error type (say `InvalidI32Error`) if and only if the latter's detail record type (`InvalidIntDetail`) is super type of the former's detail type (`InvalidI32Detail`). 
```

Comment:
We shouldn't be referring to constructs from the example in the description, IMO.

Reply:
Fixed